client/app/components/OnboardingTopBar/OnboardingTopBar.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react';
+
+import css from './OnboardingTopBar.scss';
+
+class OnboardingTopBar extends React.Component {
+
+  t(translationKey) {
+    return this.props.translations[translationKey];
+  }
+
+  next() {
+    if (!this.props.status.slogan_and_description) {
+      return {
+        title: this.t('add_slogan'),
+        link: this.props.guide_root + '/slogan_and_description',
+      };
+    } else if (!this.props.status.cover_photo) {
+      return {
+        title: this.t('add_cover'),
+        link: this.props.guide_root + '/cover_photo',
+      };
+    } else if (!this.props.status.filter) {
+      return {
+        title: this.t('add_filters'),
+        link: this.props.guide_root + '/filter',
+      };
+    } else if (!this.props.status.paypal) {
+      return {
+        title: this.t('add_payments'),
+        link: this.props.guide_root + '/paypal',
+      };
+    } else if (!this.props.status.listing) {
+      return {
+        title: this.t('add_listing'),
+        link: this.props.guide_root + '/listing',
+      };
+    } else if (!this.props.status.invitation) {
+      return {
+        title: this.t('invite_users'),
+        link: this.props.guide_root + '/invitation',
+      };
+    }
+    return null;
+  }
+
+  progress() {
+    const steps = Object.keys(this.props.status).filter((entry) =>
+      ['slogan_and_description',
+       'cover_photo',
+       'filter',
+       'paypal',
+       'listing',
+       'invitation'].includes(entry)).map((key) => [key, this.props.status[key]]);
```

Comment:
Could this be spread so that next processing steps are on their own row:
```
Object.keys(this.props.status)
  .filter((entry) =>
    [ 'slogan_and_description',
      'cover_photo',
      'filter',
      'paypal',
      'listing',
      'invitation'
    ].includes(entry))
  .map((key) => [key, this.props.status[key]]);
```